src/test/App.test.tsx

```diff
@@ -69,7 +69,7 @@ describe("App", () => {
   }, 10000);
 
   it("searching terms not found displays no results", async () => {
-    await verifySearchResults(page, "gibberish", []);
```

Comment:
was this no longer passing? why would that be?

Reply:
Yeah, just the way the search algorithm is now is matching by string similarity, not pattern matching. This was matching the author's name "Gibbs". Definitely the match algorithm can be refined but it works ok in most cases.

Reply:
I see! IMO this is actually an improvement as well. Very nice.